templates/default/css/template.css

```diff
@@ -622,3 +624,47 @@ div.yes-no .btn-group label {
     border: none;
     }
 }
+
+@media only screen and (max-width: 760px){
+  h1, .h3{
+    font-size:18px;
+  }
+  
+  .h4{
+    font-size:16px;
+  }
+  
+  .question-container{
+    margin-bottom:1em;
+  }
+  
+  .question-title-container{
+    padding-top:5px;
+    padding-bottom: 5px;
+  }
+  
+  .answer-container{
+    //padding-top:0.5em;
+    //padding-bottom:0.5em;
+  }
+  
+  .ls-question-help, 
+  .ls-questionhelp{
+    font-size:14px;
+  }
+  
+  .table-bordered{
+    border:none;
+  }
+  
+  .ls-answers tr:last-child{
+    margin-bottom:0;
+  }
+  
+  .ls-answers > tbody > tr > td.radio-item ,
+  .ls-answers > tbody > tr > td.checkbox-item{
+    min-height:44px; /*minimal height for touching with finger */
+    padding-top:10px;
+    padding-left:28px;
```

Comment:
You are right, should be the same in big screen. Shal I change that? Please accept the current changes and I will do it again. (I do not know how do retract commits...)